src/main/java/com/google/cloud/spanner/pgadapter/utils/PgJdbcCatalog.java

```diff
@@ -957,4 +957,9 @@ public class PgJdbcCatalog {
           + "inner join pg_catalog.pg_class rel on a.attrelid=rel.oid\n"
           + "inner join pg_catalog.pg_namespace n on rel.relnamespace=n.oid\n"
           + "inner join information_schema.columns c on n.nspname=c.table_schema and rel.relname=c.table_name and a.attname=c.column_name\n";
+
+  public static final String PG_JDBC_GET_SEQUENCES =
+      "select relname from pg_class where relkind='S'";
+  public static final String PG_JDBC_GET_SEQUENCES_REPLACEMENT =
+      "select 1 as relname LIMIT 0";
```

Comment:
nit: Make sure the column that is returned is of type `varchar` to correspond with the return type of the actual query.

```suggestion
      "select ''::varchar as relname LIMIT 0";
```

Reply:
Done